sound/soc/sof/sof-audio.c

```diff
@@ -813,11 +813,10 @@ struct snd_sof_dai *snd_sof_find_dai(struct snd_soc_component *scomp,
 	return NULL;
 }
 
-/*
- * Helper to get SSP MCLK from a pcm_runtime.
- * Return 0 if not exist.
- */
-int sof_dai_get_mclk(struct snd_soc_pcm_runtime *rtd)
+#define SOF_DAI_CLK_MCLK	0
+#define SOF_DAI_CLK_BCLK	1
+
+static int sof_dai_get_clk(struct snd_soc_pcm_runtime *rtd, int clk_id)
```

Comment:
nit-pick: maybe clk_type to avoid confusing with the clk framework. This isn't a dynamic ID at all.

Reply:
done